config/routes.rb

```diff
@@ -26,6 +26,8 @@
     end
   end
 
+  get '/privacy', to: 'pages#privacy'
+  get '/terms', to: 'pages#terms'
```

Comment:
Might as well throw the octobox_io config conditional here, too, as the template isn't customizable.

Reply:
Good call 👍